Applications/ClientWebPortal/Source/webapp/src/utils/value-validator/index.js

```diff
@@ -0,0 +1,8 @@
+class ValueValidator {
+  isFunction(functionToCheck) {
+    var getType = {};
+    return functionToCheck && getType.toString.call(functionToCheck) === '[object Function]';
```

Comment:
getType serves no function.  Can be reduced to:

return functionToCheck && ({}).toString.call(functionToCheck) === '[object Function]';

Reply:
Ok